### ISSUE TITLE
Please provide example of creating google::protobuf::Any member

### ISSUE THREAD

**Issue by gmabey** [semlanik/qtprotobuf#167]
Hello,

I've been *extracting* data from a `google::protobuf::Any` field for several versions, and it works great thank you.

Now, I need to *populate* a Any field with `qtprotobuf`.  I understand how to serialize the target to a `QByteArray`, and then assign it with `Any::setValue()`, but I do not see a clear way to retrieve the value to be assigned with `Any::setTypeUrl()` and I don't see any appropriate (static) data members in the `message` that I wish to store.

Could you please provide an example of how to do that?

Thank you!

**Comment by semlanik** [comment]
Hi @gmabey , hope you're doing well :) I think you request functionality that I didn't implement yet. Please look at #83 . I had feeling that this functionality is not required by anyone, so I postponed it. This release I already focused on your previous requests, so will add it to next release schedule if you don't mind.

**Comment by gmabey** [comment]
Yes, what I've done for now is to just copy the string as it was produced by the python protobuf tools.  So, it's not generic, but it's working, so yes it will be fine to push that down the road.
